### DISCUSSION TITLE
lint on save

### DISCUSSION THREAD

**Discussion by pixelpaulaus** [adonisjs/core#4042] [Help]
I have a project which I initially setup without eslint.
But now I want it eslint in this project.
I have added all the dependencies and eslint values into my package file. And it works when I run `npm run lint`
But is there a way for it to lint on the watcher for when I save a file?

**Comment by RomainLanz** [comment]
This is something you have to setup inside your IDE.

**Reply by pixelpaulaus** [reply]
yeah i dont want that. I am trying to find a solution on how they do it in vite/vue

**Reply by RomainLanz** [reply]
I apologize, but I am unsure of the context or purpose of your mention of "how they do it in Vite/Vue." Could you please provide more information or clarify your question?
